MODELS

```diff
@@ -55,6 +55,10 @@ Model     MCU         Name
 0422      attiny85    fireflies-pl47-219
 0423      attiny85    fireflies-pl47g2
 0441      attiny85    fireflies-e01
+0445      attiny85    fireflies-e12c
+0451      attiny1616  fireflies-lume1-6af       (E07X, E12R, NOV-Mu, etc)
+0452      attiny1616  fireflies-lume1-6af-nofet (T9R, T1R, etc)
```

Comment:
My SBT90A T9R which I just received from FFL, blinks 0451 as model, not 0452 as you indicated.

Is there a possible divergence in model numbers between what you're using here, and FFL's actual practice?

OTOH I'm pretty sure my T9R has FET as it blinks when ramping up, just before reaching maximum brightness -- so perhaps your model number is correct and consistent with FFL after all.